environment/image_recipes/bare_metal/bare_metal.json

```diff
@@ -99,6 +100,13 @@
     {
       "type": "shell-local",
       "inline": [
+          "echo 'Resize image'",
+          "qemu-nbd -c /dev/nbd0 images/{{user `image_name`}}.qcow2",
+          "e2fsck -y -f /dev/nbd0p1",
+          "resize2fs /dev/nbd0p1 {{user `disk_size`}}G",
+          "qemu-nbd -d /dev/nbd0",
+          "qemu-img resize --shrink images/{{user `image_name`}}.qcow2 {{user `disk_size`}}G",
+
```

Comment:
More of NIT, need the extra space here?

Reply:
I separate each step with a new line to make it easier to read

Reply:
ahh okay. well carry on :)